Add department features. Scheduling rules and node pool priority to 2.19 (#1063)
run-ai/docs: docs/platform-admin/aiinitiatives/org/departments.md

When node pools are enabled, it is possible to set the above quota parameters __for each node-pool separately__.

* __Order of priority__
This column is displayed only if more than one node pool exists. The default order in which the Scheduler uses node pools to schedule a workload. 

> This means the Scheduler first tries to allocate resources using the highest priority node pool, then the next in priority, until it reaches the lowest priority node pool list, then the Scheduler starts from the highest again. The Scheduler uses the department list of prioritized node pools, only if the order of priority of node pools is not set in project or the workload during submission (either by an admin policy or by the user). Empty value means the node pool is not part of the Department’s default node pool priority list, but a node pool can still be chosen by the admin policy or a user during workload submission

Reviewer: ```suggestion
This means, the Scheduler first tries to allocate resources using the highest priority node pool, followed by the next in priority, until it reaches the lowest priority node pool list, then the Scheduler starts from the highest priority again. The Scheduler uses the department list of prioritized node pools, only if the order of priority of node pools is not set in project or the workload during submission (either by an admin policy or by the user). An empty value indicates that the node pool is not part of the department’s default node pool priority list, but a node pool can still be chosen by the admin policy or a user during workload submission.
```

> Department nodepool priority set defaults to the subordinate projects but do not enforce it, meaning projects are free to change their priority.

Reviewer: ```suggestion
Department nodepool priority sets defaults to the subordinate projects but does not enforce it, meaning projects are free to change their priority.
```
